quickfixj-examples/executor/src/main/java/quickfix/examples/executor/Application.java

```diff
@@ -315,7 +315,7 @@ public void onMessage(quickfix.fix43.NewOrderSingle order, SessionID sessionID)
         Price price = getPrice(order);
 
         quickfix.fix43.ExecutionReport accept = new quickfix.fix43.ExecutionReport(
-                    genOrderID(), genExecID(), new ExecType(ExecType.NEW), new OrdStatus(
+                    genOrderID(), genExecID(), new ExecType(ExecType.TRADE), new OrdStatus(
```

Comment:
Shouldn't this be `NEW` still? Cumulated qty is 0.

Reply:
Yes, well spotted.